### ISSUE TITLE
Setup initiatial selection of dates without triggering selectionDelegates

### ISSUE THREAD

**Issue by patchthecode** [patchthecode/JTAppleCalendar#4]
User should be able to setup initial dates without triggering selection delegates. 

Proposed fix:
selectDate function will include an optional Bool parameter to allow triggering of the delegates

**Comment by patchthecode** [comment]
Alright. Code complete. Testing now. Expect update in about 2 hrs or so.

**Comment by patchthecode** [comment]
Published 2.0.1.
Let me know if you encounter anything else.
The updated function is used like this ->
`calendarView.selectDates(dates: [NSDate], triggerSelectionDelegate: false) `
or if you want it to be true you can use it like the same way you always have->
`calendarView.selectDates(dates: [NSDate]) `
or like this ->
`calendarView.selectDates(dates: [NSDate], triggerSelectionDelegate: true) `

**Comment by antonijap** [comment]
I tested it, it works as expected. I'll let you know if I hit a bump again. Thanks.

**Comment by antonijap** [comment]
It's me again.

This new function is working in a weird way. Let me show you.

When the app first loads it runs this:
`calendarView.selectDates(selectedDays, triggerSelectionDelegate: false)`

And this works as expected, my dates get isSelected to true. Then, when I tap on a date to select it or deselect it that code is repeated, but instead of selecting every date in my array again it sets their isSelected property to false and they disappear.

I tried to play with it a little and I typed:
```
calendarView.selectDates(selectedDays, triggerSelectionDelegate: false) // makes dates in array selected
calendarView.selectDates(selectedDays, triggerSelectionDelegate: false) // this one destroys work of that previous one
```

That second function really reverts their state to the opposite of their previous state. Why is that happening?

**Comment by patchthecode** [comment]
Ok let me understand.
Tell me which part below is wrong:
1. I assume that you have multipleSelection enabled.
2. `calendarView.selectDates(selectedDays, triggerSelectionDelegate: false)`  works as expected on viewDidLoad - It correctly selects the dates.
3. After step 2, you select an already selected date (a date that was selected in the initial selection). You did this selection by tapping the simulator instead of using code. 
4. After step 3, you say that the the isSelected property goes away, and the cell's visibility dissappears?

**Comment by patchthecode** [comment]
Hey btw, any new issue, start it with a new issue. Its gonna make things harder to track if i have to search through old logs.  i have tracked this new issue here --> https://github.com/patchthecode/JTAppleCalendar/issues/6
